
time Stamp
float32[3] MagneticField # (20Hz)
                         # Three axis of magnetic field sensing in micro Tesla (uT)
